src/piWebserviceProvider.ts

```diff
@@ -130,6 +130,19 @@ export class PiWebserviceProvider {
         return res.data;
     }
 
+    /**
+     * Request time series with a relative url
+     * @param relativeUrl
+     * @returns time series api response
+     */
+    async getTimeSeriesWithRelativeUrl(relativeUrl: string): Promise<TimeSeriesResponse> {
+        const requestInit = {} as RequestInit;
+        requestInit.cache = "no-cache";
+        const urlString = this.baseUrl + "/" + relativeUrl;
+        const url = new URL(urlString);
```

Comment:
You can combine the URL constructor with combining the base and relative URLs:

```typescript
const url = new URL(relativeUrl, this.baseUrl);
```
I prefer this method over adding your own slashes.

Reply:
done